src/rust/src/series/aggregation.rs

```diff
@@ -0,0 +1,111 @@
+use crate::{PlRSeries, RPolarsErr, Wrap};
+use polars::datatypes::DataType;
+use polars::prelude::{QuantileMethod, Scalar};
+use polars_core::prelude::AnyValue;
+use savvy::{NumericScalar, Result, Sexp, savvy};
+
+fn scalar_to_r(x: Scalar) -> Result<Sexp> {
+    match x.as_any_value() {
+        AnyValue::Int8(x) => return (x as i32).try_into(),
+        AnyValue::Int16(x) => return (x as i32).try_into(),
+        AnyValue::Int32(x) => return x.try_into(),
+        AnyValue::Int64(x) => return (x as f64).try_into(),
```

Comment:
`product()` returns i64 but I don't think we want to add the arg `int64_conversion` in this function only (but we could).